### ISSUE TITLE
Remove appveyor

### ISSUE THREAD

**Issue by MartinSStewart** [stil4m/elm-syntax#187]
Appveyor fails on every build and seems difficult to fix https://github.com/stil4m/elm-syntax/pull/106

Also it seems redundant since we have CI that runs tests and publishes already. Can we just remove appveyor instead?

**Comment by jfmengels** [comment]
I agree. Unfortunately, I think that happens in the project's Settings, and I don't have access to that.

@stil4m would it be possible to grant us full rights to the project? And then we can probably be able to fly on our own and we won't bother you again :sweat_smile: :crossed_fingers: 
(I'm not sure whether that'll be sufficient for appveyor though)
